target/linux/ath79/image/generic.mk

```diff
@@ -43,6 +43,28 @@ define Build/elecom-header
 
 endef
 
+define Device/seama
```

Comment:
@hackpascal Device support belongs to the other commit.

Reply:
I'll re-organize it.